scripts/jenkins/ardana/ansible/templates/dac-3cp/input-model-servers.yml

```diff
@@ -0,0 +1,49 @@
+---
+  product:
+    version: 2
+
+  baremetal:
+    subnet: 192.168.245.0
+    netmask: 255.255.255.0
+
+  servers:
+
+    - id: cp1-0001
+      ip-addr: {{ controller_mgmt_ips[0] }}
+      role: STD-CONTROLLER-ROLE
+      server-group: RACK1
+      mac-addr: b2:72:8d:ac:7c:6f
+      ilo-ip: 192.168.109.3
+      ilo-password: password
+      ilo-user: admin
```

Comment:
I guess that this comes from the original input model, or is needed? If not maybe we can drop all the ilo information

Reply:
I think it's still needed for config processor validation purposes, but I'm not an expert. I only followed what seemed to be the common approach.

Reply:
Uhm, I can be wrong but I think that is not the case:

https://github.com/ArdanaCLM/ardana-configuration-processor/blob/master/ConfigurationProcessor/ardana_configurationprocessor/plugins/validators/2_0/ServersValidator.py#L122